contribs/gmf/src/editing/Snapping.js

```diff
@@ -553,7 +553,18 @@ EditingSnappingService.prototype.loadItemFeatures_ = function(item) {
 
       // (3) Read features from request response and add them to the item
       const readFeatures = new olFormatWFS().readFeatures(response.data);
+      const me = this;
```

Comment:
Please remove this line and use 
```
const myFn = fn(){}
myFn.bind(this);
```

Or use arrow function (and too bad for the coherency)